### PR TITLE
WT-6229 add javadoc

### PR DESCRIPTION
**Pull request by sukolenvo**

### CODE REVIEW

File: src/main/java/io/wisetime/connector/IntegrateApplication.java [hunk 1]
```diff
@@ -21,7 +21,10 @@
 import static spark.Spark.staticFileLocation;
 
 /**
+ * Spark application that handles time posted event and basic UI.
```
Comment (vyshane) [inline src/main/java/io/wisetime/connector/IntegrateApplication.java]: Do we have a UI for the connector?

**Comment by vyshane** [comment]
Looks good, thanks for doing this. I've pushed some small tweaks to the docs - faster that way than going back and forth.

File: src/main/java/io/wisetime/connector/ServerRunner.java [hunk 2]
```diff
@@ -182,16 +217,34 @@ private ConnectorStore createStore(boolean persistenceRequired) {
...
 
+    /**
+     * Whether persistent storage is required for this connector. If true, this property forces the operator to set the
+     * DATA_DIR configuration parameter (via system property or environment variable). The DATA_DIR directory will be used
+     * to persist data across connector restarts.
+     *
+     * If false, the connector will not force the operator to specify a DATA_DIR. It will still use the DATA_DIR path if
+     * one is set. If none is provided, a subdirectory in the /tmp/ path will be used.
+     *
+     * Default is false.
```
Comment (vyshane) [inline src/main/java/io/wisetime/connector/ServerRunner.java]: @sukolenvo I've taken the liberty to update the docs above. Can you please check that my understanding is correct?

**Comment by vyshane** [comment]
Please merge if [this comment](
https://github.com/wisetime-io/wisetime-connector-java/pull/17#discussion_r244077496) is correct.

File: src/main/java/io/wisetime/connector/IntegrateApplication.java (hunk 1 above)
Comment (sukolenvo) [reply src/main/java/io/wisetime/connector/IntegrateApplication.java]: Its just single page that says "WiseTime connector is running"

**Pull request merged**